packages/react/src/SegmentedControl/SegmentedControlButton.tsx

```diff
@@ -31,43 +21,35 @@ export type SegmentedControlButtonProps = {
 } & SxProp &
   ButtonHTMLAttributes<HTMLButtonElement | HTMLLIElement>
 
-const SegmentedControlButtonStyled = toggleStyledComponent(
-  SEGMENTED_CONTROL_CSS_MODULES_FEATURE_FLAG,
-  'button',
-  styled.button`
-    ${getGlobalFocusStyles('-1px')};
-    ${sx};
-  `,
-)
-
 const SegmentedControlButton: React.FC<React.PropsWithChildren<SegmentedControlButtonProps>> = ({
   children,
   leadingIcon: LeadingIcon,
   selected,
   sx: sxProp = defaultSxProp,
   className,
+  // Note: this value is read in the `SegmentedControl` component to determine which button is selected but we do not need to apply it to an underlying element
+  defaultSelected: _defaultSelected,
   ...rest
 }) => {
-  const enabled = useFeatureFlag(SEGMENTED_CONTROL_CSS_MODULES_FEATURE_FLAG)
-  const mergedSx = enabled ? sxProp : merge(getSegmentedControlListItemStyles(), sxProp as SxProp)
-
   return (
-    <Box as="li" sx={mergedSx} className={clsx(enabled && classes.Item)} data-selected={selected || undefined}>
-      <SegmentedControlButtonStyled
+    <BoxWithFallback as="li" sx={sxProp} className={clsx(classes.Item)} data-selected={selected || undefined}>
```

Comment:
```suggestion
    <BoxWithFallback as="li" sx={sxProp} className={clsx(classes.Item)} data-selected={selected ? '' : undefined}>
```

For the boolean-ish attributes we've been needing to do this so that it formats to `data-selected` versus `data-selected="true|false"`.